scraper.rb

```diff
@@ -25,20 +28,12 @@ def scrape_list(url)
   noko = noko_for(url)
   puts url.to_s.yellow
 
-  noko.css('ul.lisitng_resultat li').each do |li|
-    data = { 
-      id: li.css('h2.name a/@href').text.split("/").last,
-      name: li.css('h2.name').text.tidy,
-      party: li.css('a.link span').first.text,
-      faction: li.css('a.link span').last.text,
-      source: li.css('h2.name a/@href').text,
-      image: li.css('a.figure img/@src').text,
-      term: 9,
-    }
-    data[:name] = 'Kamal Abdel Fattah' if data[:id] == 'akamal' # No name in English version
-    data[:source] = URI.join(url, URI.escape(data[:source])).to_s unless data[:source].to_s.empty?
-    ScraperWiki.save_sqlite([:id, :term], data) unless data[:name] == 'Vaccant Poste'
-  end
+  MembersPage.new(response: Scraped::Request.new(url: url).response(decorators: [AbsoluteLinks]), noko: noko)
+             .members
+             .each do |member|
+                puts member
```

Comment:
this is a slightly odd thing to be doing here. There's no `to_s` defined on these, so this just outputs things like `#<MemberSection:0x007fef039f4710>`, which is less than helpful. I doubt you actually want to be outputting anything at all though in production use, as unnecessary console output makes morph quite unhappy.